static/app/components/events/eventStatisticalDetector/spanDiff.tsx

```diff
@@ -0,0 +1,227 @@
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import EmptyStateWarning from 'sentry/components/emptyStateWarning';
+import {DataSection} from 'sentry/components/events/styles';
+import Link from 'sentry/components/links/link';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import TextOverflow from 'sentry/components/textOverflow';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconAdd, IconSubtract} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {defined} from 'sentry/utils';
+import {getDuration} from 'sentry/utils/formatters';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {spanDetailsRouteWithQuery} from 'sentry/views/performance/transactionSummary/transactionSpans/spanDetails/utils';
+
+type SpanDiff = {
```

Comment:
I remember at some point we wanted to start using `interface` instead of a `type` across the repo